### ISSUE TITLE
Fix bug with linking associations

### ISSUE THREAD

**Issue by sdegutis** [Atlas-Authority/marketing-automation#101]
With what's most likely corrupted data due to Atlassian, linking associations can fail with the following stacktrace:

```
4/18/2022, 9:50:37 PM ERR! Runner Error:
4/18/2022, 9:50:37 PM ERR! Runner   TypeError: Cannot read properties of undefined (reading 'get')
4/18/2022, 9:50:37 PM ERR! Runner       at DealManager.linkEntities (/usr/src/app/src/lib/hubspot/manager.ts:60:50)
4/18/2022, 9:50:37 PM ERR! Runner       at Hubspot.importData (/usr/src/app/src/lib/hubspot/hubspot.ts:34:22)
4/18/2022, 9:50:37 PM ERR! Runner       at new DataSet (/usr/src/app/src/lib/data/set.ts:38:18)
4/18/2022, 9:50:37 PM ERR! Runner       at DataManager.dataSetFrom (/usr/src/app/src/lib/data/manager.ts:52:21)
4/18/2022, 9:50:37 PM ERR! Runner       at work (/usr/src/app/src/bin/main.ts:30:33)
4/18/2022, 9:50:37 PM ERR! Runner       at Timeout.run [as _onTimeout] (/usr/src/app/src/lib/util/runner.ts:24:7)
4/18/2022, 9:50:37 PM WARN Runner Run canceled by error. Trying again in 4h.
4/18/2022, 9:50:37 PM info Slack Failed 3 times. Below are the specific errors, in order. Trying again in 4h.
```

**Comment by sdegutis** [comment]
This apparently happens because sometimes an entity kind can be "company_unlabeled" instead of just "company". Not sure why. Quick fix (sending PR) is to just remove "_unlabeled" from it, which gets the engine working again. Longer term, we need to know what this type is and why HubSpot is giving it to us sometimes. Googled it and found nothing.

**Comment by sdegutis** [comment]
Fixed by #102
